validator/src/main/java/com/amazon/aoc/fileconfigs/PredefinedExpectedTemplate.java

```diff
@@ -222,23 +222,26 @@ public enum PredefinedExpectedTemplate implements FileConfig {
 
   /** Python EC2 ADOT SigV4 (Stand Alone ADOT) Test Case Validations */
   PYTHON_EC2_ADOT_SIGV4_OUTGOING_HTTP_CALL_LOG(
-      "/expected-data-template/python/ec2/adot-sigv4/outgoing-http-call-log.mustache"),
+      "/expected-data-template/python/ec2/adot-sigv4/traces/outgoing-http-call-log.mustache"),
```

Comment:
```suggestion
      "/expected-data-template/python/ec2/adot-aws-otlp/traces/outgoing-http-call-log.mustache"),
```

It would be nice if we can change all the `sigv4` references to `aws-otlp`. I know this was done previously for traces and you are simply following the pattern, but I think `aws-otlp` makes more sense. It's non-blocking though.

Reply:
Edited this as well